app/pages/project/instances/instance/tabs/NetworkingTab.tsx

```diff
@@ -371,7 +372,7 @@ export function Component() {
 
   const ipTableInstance = useReactTable({
     columns: useColsWithActions(staticIpCols, makeIpActions),
-    data: eips?.items || [],
+    data: orderIps(eips.items),
```

Comment:
I think you need to memoize this to avoid the table blowing up on every render. This would manifest, for example, as the row actions menu closing on every instance poll (which I need to put in more tests for, as mentioned in https://github.com/oxidecomputer/console/pull/2618#issuecomment-2539806844).